### PR TITLE
UKy Flowsheet Documentation

### PR DESCRIPTION
**Pull request by MarcusHolly**
## Addresses Issue: 
#55

## Summary/Motivation:
Adds documentation for the University of Kentucky flowsheet.

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.md file
   at the top level of this directory.

2. I represent I am authorized to make the contributions and grant the license. If my employer has
   rights to intellectual property that includes these contributions, I represent that I have
   received permission to make contributions and grant the required license on behalf of that
   employer.

### CODE REVIEW

**Comment by MarcusHolly** [comment]
I created this initial draft of the UKy flowsheet documentation. I've tried to model it after how we are documenting the flowsheets in WaterTAP, but I'm open to suggestions for improving it

**Comment by andrewlee94** [comment]
@MarcusHolly Did you talk to @lbianchi-lbl about how to do the docs here? From talking to him about the leaching docs, we are trying to put most/all of the docs in the module doc strings rather than having separate .rst files for them.

**Comment by MarcusHolly** [comment]
Oh, alright, so just make use of the functionality that was merged in with #56? I feel like the doc strings would need to be quite lengthy in that case (at least much more sophisticated than the ones I've added in this PR thus far)

**Comment by andrewlee94** [comment]
@MarcusHolly Yes - I see nothing wrong with long doc strings.

File: src/prommis/uky/uky_flowsheet.py [hunk 1]
```diff
@@ -10,10 +10,138 @@
...
+===================================================================== ============ ============================
+Leaching
+Tank volume                                                           100          :math:`\text{m}^3`
+Liquid feed volumetric flow                                           224.3        :math:`\text{L/hr}`
+Liquid feed H concentration                                           100          :math:`\text{mg/L}`
+Liquid feed HSO4 concentration                                        1e-8         :math:`\text{mg/L}`
+Liquid feed SO4 concentration                                         4800         :math:`\text{mg/L}`
+Liquid feed REE and contaminant concentrations                        1e-10        :math:`\text{mg/L}`
+Solid feed mass flow                                                  22.68        :math:`\text{kg/hr}`
+Solid feed inerts mass fraction                                       0.6952       :math:`\text{dimensionless}`
```
Comment (andrewlee94) [inline src/prommis/uky/uky_flowsheet.py:73]: Are the feed mass fractions correct? I see that many are the same, and it looks like they would not sum to 1.

**Review by andrewlee94 — COMMENTED** [review]
I see one thing that I think need to be corrected, but otherwise looks good.

File: src/prommis/uky/uky_flowsheet.py (hunk 1 above)
Comment (MarcusHolly) [reply src/prommis/uky/uky_flowsheet.py:73]: Good catch - I need to correct this.

**Review by agarciadiego — APPROVED** [review]
LGTM

**Review by andrewlee94 — APPROVED** [review]

**Pull request merged**